tasks/src/image-to-image/data.ts

```diff
@@ -54,7 +54,16 @@ const taskData: TaskDataCustom = {
 			id:          "lambdalabs/sd-image-variations-diffusers",
 		},
 	],
-	spaces:       [],
+	spaces:       [
```

Comment:
Maybe https://huggingface.co/spaces/huggingface-projects/diffuse-the-rest would also be a good fit here, as it has been one of the most popular demos .